.github/workflows/foundry.yml

```diff
@@ -21,5 +21,8 @@ jobs:
       - name: Install Foundry
         uses: foundry-rs/foundry-toolchain@v1
 
-      - name: Test
+      - name: Check contract sizes
```

Comment:
Might also be worth adding `forge coverage` to the ci as well. right now forge coverage is throwing a compile error, but if we add the line `src/=lib/optimism/packages/contracts-bedrock/src/` to remappings.txt file, its hould fix it (looks like it's some bug in the optimism library)

Reply:
(thanks to @Inkvi  for the suggestion)